cg/resources/raredisease_bundle_filenames.yaml

```diff
@@ -0,0 +1,965 @@
+- format: cram
+  id: SAMPLEID
+  path: PATHTOCASE/alignment/SAMPLEID_sorted_md.cram
+  path_index: PATHTOCASE/alignment/SAMPLEID_sorted_md.cram.crai
+  step: alignment
+  tag: ~
+- format: cram
+  id: SAMPLEID
+  path: PATHTOCASE/alignment/SAMPLEID_mt_subsample.cram
+  path_index: PATHTOCASE/alignment/SAMPLEID_mt_subsample.cram.crai
+  step: alignment
+  tag: ~
+- format: txt
+  id: SAMPLEID
+  path: PATHTOCASE/alignment/SAMPLEID_sorted_md.MarkDuplicates.metrics.txt
+  path_index: ~
+  step: alignment
+  tag: ~
+- format: bw
+  id: SAMPLEID
+  path: PATHTOCASE/qc_bam/SAMPLEID_qualimap/SAMPLEID_tidditcov.bw
+  path_index: ~
+  step: tiddit_coverage
+  tag: ~
+- format: png
+  id: SAMPLEID
+  path: PATHTOCASE/qc_bam/SAMPLEID_chromographcov/SAMPLEID_tidditcov_1.png
+  path_index: ~
+  step: chromograph_cov
+  tag: tcov
+- format: png
+  id: SAMPLEID
+  path: PATHTOCASE/qc_bam/SAMPLEID_chromographcov/SAMPLEID_tidditcov_2.png
+  path_index: ~
+  step: chromograph_cov
+  tag: tcov
```

Comment:
All these files should have unique tags so that each file can be distinguished from others through its specific tag/tag sets